src/core.js

```diff
@@ -510,6 +509,15 @@ $.extend( $.validator, {
 			return this.size() === 0;
 		},
 
+		// Check if the given element is valid
+		// return
+		//          true  If the field is valid
+		//         false  If the field is invalid
+		//     undefined  If the field is not validated yet.
+		isValidElement: function( element ) {
```

Comment:
having another look at this method...

it assumes that `validate()` was called before, right? I think we should at least mention this precondition in the method doc.